### DISCUSSION TITLE
virtme-ng-init  --version ?

### DISCUSSION THREAD

**Discussion by jimc** [arighi/virtme-ng#133] [Ideas]
```
[jimc@frodo virtme-ng.git]$ virtme/guest/bin/virtme-ng-init 
virtme-ng-init: must be run as PID 1
```

Could this invocation also report info, like --version might ?

**Comment by arighi** [comment]
Well... virtme-ng-init doesn't support command lines options (not yet at least), maybe we having `--version` could be useful to debug boot issues, so that we can check the exact version of virtme-ng-init that it's used. I'll add that to my todo list. Thanks!

**Reply by jimc** [reply]
it would be bells and whistles,
but a json dump to stdout,
with qemu / kvm version info as well,
would be slick.
And since PID != 1 ends abruptly,
it wouldnt break any existing behavior.
It doesnt actually need any --version option to do this.
